lib/cache.js

```diff
@@ -51,14 +48,13 @@ Cache.prototype.clr = function(
   });
 };
 
-const cache = (
-  // Extend Map interface with:
-  // `cache.allocated` - total allocated size
-  // `cache.add(key, val)` - add record
-  // `cache.del(key)` - delete record
-  // `cache.clr(prefix, fn)` - delete all if `key.startsWith(prefix)`
-  // Returns: object, cache instance
-) => (new Cache());
+// Extend Map interface with:
```

Comment:
We can probably replace this whole comment with just
Returns new Cache instance
Or something? As all of the explanation will be present in the Cache class documentation.